src/utils/userLocation.ts

```diff
@@ -59,46 +54,16 @@ export const updateCurrentLocationData = ({
   userId,
   profileLocationData,
 }: UpdateCurrentLocationDataProps) => {
-  const [locationName] = Object.keys(profileLocationData);
-
   updateLocationData({
     userId,
-    newLocationData: { [locationName]: getCurrentTimeInMilliseconds() },
+    newLocationPath: profileLocationData,
   });
 };
 
 // TODO: refactor how user location updates works here?
 //   Called from VenuePage useEffect + onBeforeUnloadHandler
 export const clearLocationData = (userId: string) => {
-  updateLocationData({ userId, newLocationData: {} });
-};
-
-export interface EnterExternalRoomProps {
-  userId: string;
-  roomUrl: string;
-  locationName: string;
-}
-
-export const enterExternalRoom = ({
```

Comment:
Just making sure I understand why we don't need this anymore - is it because all external rooms links are opened from a dedicated venue page?

Reply:
Yes, correct